src/mumble/main.cpp

```diff
@@ -276,6 +278,16 @@ int main(int argc, char **argv) {
 								   "  --print-echocancel-queue\n"
 								   "                Print on stdout the echo cancellation queue state\n"
 								   "                (useful for debugging purposes)\n"
+								   " --print-translation-dirs\n"
```

Comment:
Misalignment.

Reply:
With respect to what?
I think the alignment seems just fine :thinking:

Reply:
Ah nvm. I just noticed the other options have 2 leading spaces :point_up: